libs/coin-modules/coin-ton/src/utils.ts

```diff
@@ -118,3 +195,108 @@ export const getLedgerTonPath = (path: string): number[] => {
   }
   return numPath;
 };
+
+/**
+ * Checks if the given payload is a jetton transfer.
+ */
+export const isJettonTransfer = (payload: TonPayloadFormat): boolean =>
```

Comment:
the "isJettonTransfer" method is not used, right?

Reply:
No, the function is used in a test file, I'll move it into that test file. Thank you